packages/components/loading-spinner/src/loading-spinner.visualroute.js

```diff
@@ -5,7 +5,7 @@ export const routePath = '/loading-spinner';
 
 export const component = () => (
   <Suite>
-    <Spec label='with scale "l" (default)'>
+    <Spec label='with scale "l", maxDelayDuration "1000" (default)'>
```

Comment:
looks like I need to do more to account for the timeout here.  I will look into this.

Reply:
The percy/pupeteer tests now pass w/the loading spinner showing up after adding a `waitFor` method to the visualspec.  The jest/unit tests test to ensure that the spinner does not display if the component has been mounted for less than the time period set by the `maxDelayDuration` prop, so I am unsure whether I should also test that behavior in the e2e tests.  Please let me know what the best practice would be for a case like this.

 Either way, the tests on percy will need to reviewed/approved since the new prop was added.